libs/safekeeper_api/src/send_wal.rs

```diff
@@ -0,0 +1,2 @@
+//! This module contains structs used in safekeeper send_wal.rs which are also
+//! exposed in http API.
```

Comment:
These two files seem to be empty. Is this intended?

Reply:
No, forgot to remove, fixed. I was thinking to add more modules, but then decided it is not very useful.